R/matrix_form.R

```diff
@@ -51,14 +55,14 @@ mform_handle_newlines <- function(matform) {
         )
       }
       newstrmat[1:new_nlines_hdr, 1] <- c(tl, rep("", new_nlines_hdr - length(tl)))
-      newfrmmat[1:new_nlines_hdr, 1] <- "xx"
+      newfrmmat[1:new_nlines_hdr, 1] <- "xx" # Why is this needed?
```

Comment:
@gmbecker Is it needed?

Reply:
This is setting the "format" for the topleft area to `"xx"` (mostly so that it cannot possibly have a different format, I think). I agree that its odd but its unrelated to the work at hand so lets leave it for now and we an do a deeper investigation once the fire is put out.